script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_1_3.txt

```diff
@@ -50,19 +50,19 @@ Anyway, I come back to my senses.
 I clear my throat to change the atmosphere.
 <Page15>
 "I'm sorry, I got a little distracted.
-More importantly, is something the matter? Senpai, you're not a member of any clubs. Why are you heading to school this early?"
+More importantly, is something the matter? Senpai, you're not a member of any clubs. Why are you heading to school this early?"//similar to 01_00_ARC01_1_2 at this point refer to that file when editing
 <Page16>
-"Well, what do I have to hide? I'm on my way back from volunteer activities.
+"Well, to tell you the truth, I'm on my way back from volunteer activities.
 You see, the student council asked for volunteers to pick up trash in the park, right? That's what I was doing until just now."
 <Page17>
 When she says that, I let out an 'Ah' as everything slides into place.
 Our attractively bespectacled Senpai is always working hard for the sake of others.
 <Page18>
-"It was just an early morning volunteer activity for 30 minutes.
+"It was just an early morning volunteer activity that went for 30 minutes.
```

Comment:
"that went on for 30 mins"

Reply:
> 早朝から30分だけの奉仕活動だったんですけどね
It was just a thirty minute early morning volunteer activity.